test/integration/integration_test.go

```diff
@@ -181,6 +196,72 @@ func ensureTestDirEmpty() {
 	}
 }
 
+func setProxy() error {
+	go startProxy()
+	ip := getHostIP()
+	if ip == nil {
+		return errors.New("Not found any interface with IP and MAC.")
+	}
+	address := fmt.Sprintf("http://%v:3128", ip.String())
+	os.Setenv("MINISHIFT_HTTP_PROXY", address)
+	return nil
+}
+
+func unsetProxy() error {
+	os.Unsetenv("MINISHIFT_HTTP_PROXY")
+	if proxyListener != nil {
+		proxyListener.Close()
+	}
+	return nil
+}
+
+func startProxy() error {
+	var err error
+	proxyListener, err = net.Listen("tcp", ":3128")
+	if err != nil {
+		return err
+	}
+	proxy := goproxy.NewProxyHttpServer()
+	proxy.Verbose = true
+	proxyLog = bytes.NewBufferString("proxy log:\n")
+	proxy.Logger = log.New(proxyLog, "", log.Ldate)
+	http.Serve(proxyListener, proxy)
+	return nil
+}
+
+func getHostIP() net.IP {
```

Comment:
@gbraad Having some weak moment now, so I would like to ask you if I am on right track or completely lost:

In order to try func HostIPs() from file /pkg/minishift/util/hostip.go
I import the package util as minishiftUtil:
```golang
import (
  ...
  "github.com/minishift/minishift/test/integration/util"
  minishiftUtil "github.com/minishift/minishift/pkg/util"
  ...
)
```
Then hostIPs() should be accessible with: minishiftUtil.hostIPs()
```golang
ips := minishiftUtil.HostIPs()
```
But I am getting
```
test/integration/integration_test.go:205: undefined: "github.com/minishift/minishift/pkg/util".HostIPs
```
with which I am not sure what to do.